blosc/blosc2.c

```diff
@@ -1667,6 +1672,9 @@ static int do_job(blosc2_context* context) {
       free_thread_context(context->serial_context);
       context->serial_context = create_thread_context(context, 0);
     }
+    if (context->serial_context == NULL) {
+      return BLOSC2_ERROR_THREAD_CREATE;
+    }
```

Comment:
Couldn't this be just a single `BLOSC_ERROR_NULL` line?

Reply:
Fixed.